docs/dev/Kibana-Reporting-Design-Proposal.md

```diff
@@ -50,7 +50,34 @@ Kibana Reports for Open Distro allows ‘Report Owner’ (engineers, including b
         * As a user, I should be able to archive reports. 
             * Delete option should not be there as it might lead to complications such as one wherein the user who receives the reports deletes them and says that they never received the reports
         * As a user, I should be able to see a list of reports archived for the past 100 days. NOTE: Reports will be stored as hyperlinks/urls and NOT as actual reports. If the data is available at the time when the url is clicked, report will be created and can be viewed in a browser and can be downloaded.
-
+    * For Saved Search CSV Report
+        * As a user, I should be able to find my saved search through an autocomplete input, in order to find my saved search quickly.
+        * As a user, I should be able to choose time range for my csv data generation. This time range should be by default the same value of previous chosen time range from other tabs
+        * As a user, when I click on ‘Generate’ button, reporting plugin should launch the csv generation
+        * As a user, I should have the list of my generated csv files
+            * As a user, I want to see the name of the user who generated the file in the generated csv files list
```

Comment:
Does this mean a user would be able to see the list of all users who generated CSV reports? I'm not clear on what this means

Reply:
Good point, users should only see their generated CSV reports

```suggestion
```